hq_superset/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
nit: we need this file?

Reply:
It was generated by Alembic. We don't really need it. 0974df5